.pre-commit-config.yaml

```
@@ -30,6 +30,6 @@ repos:
     rev: 24.10.0
     hooks:
       - id: black
-        additional_dependencies: ['.[jupyter]']
-        types: [python, jupyter]
-        types_or: [python, jupyter]
+        types: [python]
+      - id: black-jupyter
+        types: [jupyter]
```

```suggestion
      - id: black-jupyter
```

You only need to use `black-jupyter`. It defines `types_or: [python, pyi, jupyter]`.